.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@
 #
 repos:
```

Comment:
We should remove these changes from the PR before merging! Not sure if they are already in the main repo or if they were the product of running pre-commit autoupdate or so.

Reply:
👍, I would favour limiting the scope of the PR to `PDBList`.
It would be a good idea to update the pre-commit dependencies version, but it can be done in a dedicated PR.

Reply:
You can easily revert this commit (40b0c48d392db0dd3da3ac6c44ae350191742008).
To do so: `git revert 40b0c48d392db0dd3da3ac6c44ae350191742008`.
Notice, that the commit contains also a `print` update, that could be re-apply individually.
_(If you want to challenge your Git skills choosing the best pratice, have a look at the [interactive rebase](https://jwiegley.github.io/git-from-the-bottom-up/1-Repository/8-interactive-rebasing.html))_